docs/rest_api_2.rst

```diff
@@ -0,0 +1,4156 @@
+REST API
+========
+
+This section will go over the specifies of the REST APIs supported by F5 ACI ServiceCenter application.
```

Comment:
This section contains the specifics of the REST APIs supported by the F5 ACI ServiceCenter application.

Reply:
Resolved in docs/rest_api.rst (docs/rest_api_ep.rst  is an outdated filename)